spec/order_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'order'
+
+describe Order do
+  let(:menu_list)      { [
+                          { dish: 'biryani', price: 10, available?: true }, 
+                          { dish: 'chicken', price: 15, available?: true },
+                          { dish: 'lamb', price: 20, available?: false },
+                          { dish: 'duck', price: 25, available?: false },
+                         ]
+                        }
+  let(:menu)           { double :menu, menu_list: menu_list }
+  let(:subject)        { described_class.new(menu) }
```

Comment:
a clever thing to set up, it simulates a menu instead of hard coding it into the production code